components/resources/jobExplorer/jobList/JobListComponent.tsx

```diff
@@ -10,14 +16,12 @@ const JobGroup = ({ subheading, jobs }) => (
         <div
           key={job.title}
           className={`flex flex-col items-center rounded-lg p-6 shadow-md ${
-            (job.title.includes("Game Developer") ||
-              job.title.includes("Cyber Security Analyst") ||
+            ( job.title.includes("Cyber Security Analyst") ||
               job.title.includes("Product Manager") ||
               job.title.includes("Digital Marketer")) &&
             "bg-gray-300 text-black"
           } ${
-            !(job.title.includes("Game Developer") ||
-              job.title.includes("Cyber Security Analyst") ||
+            !(job.title.includes("Cyber Security Analyst") ||
```

Comment:
same here. Don't Repeat Yourself. DRY Principle

https://www.geeksforgeeks.org/dont-repeat-yourselfdry-in-software-development/

Reply:
I'll make sure to reference the disabled jobs array here so I don't repeat myself